services/QuillLMS/db/migrate/20251017153523_drop_unused_indexes.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class DropUnusedIndexes < ActiveRecord::Migration[7.1]
+  def change
+    remove_index :users, :time_zone
```

Comment:
`users` is a large table (as are some of the other ones) so I believe we need to do something like  this to make sure Postgres doesn't lock the whole table (worth googling 'Rails Postgres concurrent drop index no table lock' or something like that.
```ruby    
class RemoveIndexFromTableName < ActiveRecord::Migration[7.1]
  disable_ddl_transaction! # Required for concurrent index operations

  def change
    remove_index :table_name, name: :index_name, algorithm: :concurrently
   end
 end
```